packages/components/src/TableToolsTable/TableToolsTable.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import { Pagination, PaginationVariant } from '@patternfly/react-core';
+import { Table, TableBody, TableHeader } from '@patternfly/react-table';
+import PrimaryToolbar from '../PrimaryToolbar';
+import TableToolbar from '../TableToolbar';
+import useTableTools from '@redhat-cloud-services/frontend-components-utilities/useTableTools';
+
+const TableToolsTable = ({
+  items = [],
+  columns = [],
+  filters = [],
+  options = {},
+  toolbarProps: toolbarPropsProp,
+  tableHeaderProps,
+  tableBodyProps,
+  tableToolbarProps,
+  paginationProps,
+  columnManagerProps,
+  ...tablePropsRest
+}) => {
+  const { toolbarProps, tableProps, ColumnManager } = useTableTools(items, columns, {
+    filters,
+    toolbarProps: toolbarPropsProp,
+    tableProps: tablePropsRest,
+    ...options,
+  });
+
+  return (
+    <React.Fragment>
+      <PrimaryToolbar {...toolbarProps} />
+
+      <Table {...tableProps}>
+        <TableHeader {...tableHeaderProps} />
+        <TableBody {...tableBodyProps} />
+      </Table>
+
+      <TableToolbar isFooter {...tableToolbarProps}>
+        <Pagination variant={PaginationVariant.bottom} {...toolbarProps.pagination} {...paginationProps} />
+      </TableToolbar>
+
+      {ColumnManager && <ColumnManager {...columnManagerProps} />}
```

Comment:
I hear about this component for the first time. What is it used for?

Reply:
This allows for, well, managing columns of tables. :)

https://user-images.githubusercontent.com/7757/182338632-21a983f0-151f-4fc0-9e39-e8499cc8d3dc.mp4

Reply:
Cool stuff, thanks